metrics/mixin/alerts/perf.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ODF-ceph-mds-high-cpu-warnings.rules',
+        rules: [
+          {
+            alert: 'MDS-high-cpu',
+            expr: |||
+              pod:container_cpu_usage:sum{%(mdsSelector)s}/ on(pod) kube_pod_resource_limit{resource='cpu',%(mdsSelector)s} > 0.67
+            ||| % $._config,
+            'for': $._config.mds_cpu_usage_high_threshold_duration,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Ceph metadata server pod ({{ $labels.pod }}) has high cpu usage',
+              description: 'Ceph metadata server pod ({{ $labels.pod }}) has high cpu usage.\nPlease consider increasing the number of active metadata servers,\nit can be done by increasing the number of activeMetadataServers parameter in the StorageCluster CR.',
+              severity_level: 'warning',
```

Comment:
It is needed to add  the `runbook__url `annotation with the link to the alert documentation file in runbooks repo.

Reply:
@manishym This comment still needs to be addressed.

Reply:
@umangachapagain It requires another PR in another repo, I will address it after the feature freeze.